src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'm'
+          ? 'wife'
+          : 'husband'}`
+        : 'I am not married'}
```

Comment:
Try to avoid nested condition, you can create variable like 
```
const marriedStatus = `${person.partnerName} is my ${person.sex === 'm'  ? 'wife' : 'husband'}`
```
 and use it in your JSX

Reply:
Thank you for the review!